hack/tools/release/notes.go

```diff
@@ -186,9 +192,9 @@ func run() int {
 	}
 
 	fmt.Printf("The image for this release is: %v\n", latestTag)
-	fmt.Printf("Ironic release is capm3-%v\n", latestTag)
-	fmt.Printf("Mariadb release is capm3-%v\n", latestTag)
-	fmt.Println("_Thanks to all our contributors!_ 😊")
+	fmt.Printf("Ironic image tag is capm3-%v\n", latestTag)
+	fmt.Printf("Mariadb image tag is capm3-%\nv", latestTag)
```

Comment:
I guess we should fix this here since it is anyway a manual cherry pick?

Reply:
Good catch, no idea how that did not get fixed